stripe/src/main/java/com/stripe/android/view/CardInputView.java

```diff
@@ -172,6 +203,15 @@ protected void onLayout(boolean changed, int left, int top, int right, int botto
         }
     }
 
+    private void applyTint() {
```

Comment:
If we switch to having templates for all the different brands (or, more likely, using that as a style option) then we would apply tint to all the different card templates.